src/js/components/SchemaField.js

```diff
@@ -20,8 +20,8 @@ import FieldAutofocus from "#SRC/js/components/form/FieldAutofocus";
 class SchemaField extends Component {
   shouldComponentUpdate(nextProps) {
     return (
-      !deepEqual(nextProps.formData, this.props.formData, { strict: true }) ||
```

Comment:
strict told deepEqual to use `===` vs `==` for leaf nodes, which lodash does already